### PR TITLE
Added escaping for password with spaces or quotes

### PR DESCRIPTION
**Pull request by MiiRaGe**
Also added assertion in assertionless test for baseconnector tests.

### CODE REVIEW

**Comment by ZuluPro** [comment]
Olà @MiiRaGe,
Thank you very much for this PR !

Could you fix the things in comments ?

File: dbbackup/tests/test_connectors.py [hunk 1]
```diff
@@ -25,14 +26,30 @@ def test_get_connector(self):
 class BaseDBConnectorTest(TestCase):
     def test_init(self):
         connector = BaseDBConnector()
+        assert connector.database_name == DEFAULT_DB_ALIAS
```
Comment (ZuluPro) [inline dbbackup/tests/test_connectors.py]: Could you use `django.test.TestCase`'s specific methods, like `self.assertEqual` ?

File: dbbackup/tests/test_connectors.py [hunk 2]
```diff
@@ -1,7 +1,8 @@
 from __future__ import unicode_literals
 
 import os
-from mock import patch, mock_open
+from django.db import DEFAULT_DB_ALIAS
+from mock import patch, mock_open, MagicMock
```
Comment (ZuluPro) [inline dbbackup/tests/test_connectors.py]: Could you only use `mock.patch` like made here: https://github.com/MiiRaGe/django-dbbackup/blob/d33f8590f691dae46d19c31239dd546b7b8249bd/dbbackup/tests/test_connectors.py#L145

File: dbbackup/utils.py [hunk 3]
```diff
@@ -417,3 +417,13 @@ def filename_generate(extension, database_name='', servername=None, content_type
...
+
+
+def get_escaped_command_arg(arg):
+    if six.PY2:
+        import pipes
+        quote_func = pipes.quote
+    else:
+        import shlex
+        quote_func = shlex.quote
+    return quote_func(arg)
```
Comment (ZuluPro) [inline dbbackup/utils.py]: Could you move the import on top of file, with something like:
```
try:
    from pipes import quote
except ImportError:
    from shlex import quote
```
(Maybe smater ;) )

**Comment by ZuluPro** [comment]
Hello @MiiRaGe 
Sorry to ask it, but could you merge your commits into one ?

``git rebase -i 9ade13339f06d7debc5821d`

**Comment by MiiRaGe** [comment]
reduced to 1 commit, anything else ?

**Comment by ZuluPro** [comment]
@MiiRaGe No perfect, thanks a lot !

**Pull request merged**